upstairs/src/lib.rs

```diff
@@ -7178,9 +7464,10 @@ async fn up_listen(
     mut ds_reconcile_done_rx: mpsc::Receiver<Repair>,
     timeout: Option<u32>,
 ) {
-    println!("Wait for all three downstairs to come online");
+    info!(up.log, "ZZZ up_listen starts"; "source" => "up_listen");
```

Comment:
XXX?

Reply:
That's my personal `XXX`.  It means "take out or replace this debug stuff before creating a PR and confusing your co-workers"

I'll remove.